packages/cli/src/lib/ci/heroku-api.ts

```diff
@@ -0,0 +1,15 @@
+import {APIClient} from '@heroku-cli/command'
+const V3_HEADER = 'application/vnd.heroku+json; version=3'
+const PIPELINE_HEADER = `${V3_HEADER}.pipelines`
+import {Command} from '@heroku-cli/command'
+
+export async function pipelineCoupling(client: APIClient, app: string) {
+  return client.get(`/apps/${app}/pipeline-couplings`)
+}
+
+export function setConfigVars(command: Command, pipelineID: string, body: any) {
```

Comment:
This is a duplicate of packages/cli/src/lib/api.ts `setPipelineConfigVars` which just got merged in. I think I forgot to mark it in the spreadsheet.